packages/integration-sdk-cli/src/commands/compare.ts

```diff
@@ -0,0 +1,31 @@
+import { createCommand } from 'commander';
+import * as log from '../log';
+import { findDifferences } from '../utils/diff';
+
+export function compare() {
+  return createCommand('compare')
+    .storeOptionsAsProperties()
+    .arguments('<oldPath> <neoPath>')
+    .description('Compare the differences between two datasets that were downloaded from JupiterOne via the query: \n\n' +
+
+    '\t find * as e that RELATES TO as r * return e.*, r.*\n\n' +
+
+    'Example Dataset: \n\n' +
+
+    JSON.stringify(require('../utils/exampleDataset.json'), null, 4)
+    
+    , {
+      oldPath: 'relitive path to the dataset to compare against',
+      neoPath: 'relitive path to the new dataset'
+    })
+    .option(
+      '-logkey, --log-only-key-changes',
```

Comment:
Short versions of CLI args are typically a single char (e.g. `-l`) but there's nothing technically wrong with `-logkey` as the short version.